### PR TITLE
quake3e: init at 2019-09-09

### PR DESCRIPTION
**Pull request by pmiddend**
###### Motivation for this change

I wanted this package in nixpkgs.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/games/quake3/quake3e/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    owner = "ec-";
+    repo = pname;
+    rev = version;
+    sha256 = "0i9flw8h87lagdpbci6vgqkriv05p3bidgqb4pwrxls947zwfcw8";
+  };
+
+  buildInputs = [ curl libGL libX11 libXxf86dga alsaLib libXrandr libXxf86vm libXext ];
+  enableParallelBuilding = true;
+
+  patchPhase = ''
```
Comment (jonringer) [inline pkgs/games/quake3/quake3e/default.nix]: you may want to move this to postPatch, as the default patchPhase is helpful with scripts and a few other things (this probably doesn't matter as it seems to build fine).
```suggestion
  postPatch = ''
```

File: pkgs/games/quake3/quake3e/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{ stdenv, curl, libGL, libX11, libXxf86dga, alsaLib, libXrandr, libXxf86vm, libXext, fetchFromGitHub }:
+stdenv.mkDerivation rec {
```
Comment (jonringer) [inline pkgs/games/quake3/quake3e/default.nix]: please add a space :)
```suggestion

stdenv.mkDerivation rec {
```

**Comment by pmiddend** [comment]
@jonringer Thanks for the review. I've fixed your suggestions.

**Review by jonringer — APPROVED** [review]
`nix-review` passes on NixOS
diff LGTM
(I don't won quake3, so i can't test)
leaf package

**Comment by pmiddend** [comment]
Any update on this?

**Comment by jonringer** [comment]
builds fine, I would just like for someone else who owns quake, to also verify that it works :/

**Review by Lassulus — APPROVED** [review]
tested with nix-review, played a round against bots (and won)

**Pull request merged**